statics/js/components/graph-layout.js

```diff
@@ -1369,6 +1369,27 @@ TopologyGraphLayout.prototype = {
     this.simulation.nodes(nodes);
     this.simulation.force("link").links(links);
     this.simulation.alpha(1).restart();
+    this.reArrangeGraphNodes();
+  },
+
+  setFixedPosition: function(node) {
+    if(node.metadata.X && node.metadata.Y) {
+        var theNode = this.nodes[node.id];
+        if(theNode == null) return;
+        theNode.x = parseInt(node.metadata.X);
```

Comment:
I would prefer to not put "only UI specific" metadata, I would rather add something in the ui config section in order to add these kind of rules, something like

<uuid>:
  x: 100
  y: 200

Another solution could be to allow the user the change or customize the placement function by using the external JS loading mechanism.